manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -47,7 +47,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          image: quay.io/k8scsi/csi-provisioner:v1.5.0
```

Comment:
@jichenjc v1alpha1 snapshot CRDs are not supported with this new version, but our examples still use it , make sure all gets updated as well.

please recheck all the breaking changes with respect to new versions of all sidecars .

Reply:
ok, will do ,thanks for the reminder